### PR TITLE
Adding invitation to discord on feedback card

### LINKED ISSUE #3581: Show Discord invite if visitor didn't find page helpful
**Is your feature request related to a problem? Please describe.**

We currently display a FeedbackCard component at the bottom of our documentation pages, e.g. https://ethereum.org/en/developers/docs/

Image 2021-08-07 at 2 06 45 PM

If the user selects "no" we just thank them for their feedback:
Image 2021-08-07 at 2 08 07 PM

**Describe the solution you'd like**

I think we should invite them to our Discord to for additional support. e.g. "Join our public Discord and get the answer you're looking for".

Should we also consider adding this to the bottom of every page / most pages on the website?

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

### PR DESCRIPTION
**Pull request by ahmedprusevic**
## Description

Added a discord invitation link on feedback card

## Related Issue

Resolves #3581

### CODE REVIEW

**Review by wackerow — CHANGES_REQUESTED** [review]
@ahmedprusevic Thanks for the PR! Couple things... 
- We use a component named "Link" for all of our links which can replace the anchor tag with 
`<Link to="https://discord.gg/rZz26QWfCg">Discord</Link>`. I would just put this inline within the `title` constant.
- Also, this now shows "Join our public Discord and get the answer you're looking for" even if they answered "Yes" to "Did this page help answer your question?" We should only show this invite text if the user answered "No"

We already pass in `isHelpful` to the `handleClick` function, which tells us what the user chose. We should add a state variable to store that, setting it to the `isHelpful` boolean in the `handleClick` function before `setFeedbackSubmitted`. Then update `title` to include a logic check for this result. Let me know if you need a hand!

**Comment by ahmedprusevic** [comment]
> @ahmedprusevic Thanks for the PR! Couple things...
> 
> * We use a component named "Link" for all of our links which can replace the anchor tag with
>   `<Link to="https://discord.gg/rZz26QWfCg">Discord</Link>`. I would just put this inline within the `title` constant.
> * Also, this now shows "Join our public Discord and get the answer you're looking for" even if they answered "Yes" to "Did this page help answer your question?" We should only show this invite text if the user answered "No"
> 
> We already pass in `isHelpful` to the `handleClick` function, which tells us what the user chose. We should add a state variable to store that, setting it to the `isHelpful` boolean in the `handleClick` function before `setFeedbackSubmitted`. Then update `title` to include a logic check for this result. Let me know if you need a hand!

Solved, don't know if extra abstraction is needed with title or should it just be left as it is

File: src/components/FeedbackCard.js [hunk 1]
```diff
@@ -39,23 +40,34 @@ const ButtonContainer = styled.div`
...
 
-  const title = feedbackSubmitted
-    ? "Thanks for your feedback!"
-    : "Did this page help answer your question?"
+  const title = isHelpful ? (
+    <>Thanks for your feedback!</>
+  ) : (
+    <>
+      Join our public <Link to="https://discord.gg/rZz26QWfCg">Discord</Link>{" "}
+      and get the answer you're looking for
```
Comment (minimalsm) [inline src/components/FeedbackCard.js:50]: Suggested some punctuation for the negative feedback response - other than that LGTM 👍

```suggestion
      Join our public <Link to="https://discord.gg/rZz26QWfCg">Discord</Link>{" "}
      and get the answer you're looking for.
```

**Comment by minimalsm** [comment]
LGTM and works as expected. Just checking you're good with merging this as it is @wackerow?

**Review by wackerow — APPROVED** [review]
@ahmedprusevic 👍🏻 Looks good! Thanks again for helping out!

**Comment by wackerow** [comment]
@all-contributors please add @ahmedprusevic for code

**Review by jley81 — APPROVED** [review]

**Comment by samajammin** [comment]
Woohoo! Thanks @ahmedprusevic!

**Pull request merged**